app/assets/javascripts/application.js

```diff
@@ -103,10 +103,27 @@ var yesNoEnumButtons = (function () {
   }
 })()
 
+var autoAdvanceTelInputs = (function() {
+  var autoAdvance = {
+    init: function () {
+      $(".form-group > input[type='tel']").keyup(function () {
+        if (this.value.length == this.maxLength) {
```

Comment:
Small note that I prefer `===` for strict equality, since `==` does type conversion in  sometimes surprising ways.